message_sender/views.py

```diff
@@ -89,6 +90,18 @@ class InboundViewSet(viewsets.ModelViewSet):
                      'content', 'transport_name', 'transport_type',
                      'helper_metadata', 'created_at', 'updated_at',)
 
+    def create(self, request, *args, **kwargs):
+        expect = ["event_type", "in_reply_to"]
+        if set(expect).issubset(request.data.keys()):
+            if request.data['event_type'] == "close":
```

Comment:
@rudigiesler so will the `reply_to` always be empty? if so how do we determine which outbound message has the correct time for us to use for decrementing? Is the `message_id` that of the outbound message or of this incoming message?